<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="registerBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>150</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>학번:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM Hanna 11yrs Old</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LockerSpot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이름:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>비밀번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
